docs/setup/manual-installation.md

```diff
@@ -0,0 +1,217 @@
+# Manual Installation
+
+This guide shows you how to manually install and run Piri without systemd service management.
+
+## Overview
+
+Manual installation gives you full control over the Piri binary and how it runs, but requires you to:
+- Manually start/stop Piri
+- Manually update to new versions
+- Set up your own service management (if desired)
+
+**Note:** For production nodes, we recommend [Service Installation](./service-installation.md) instead, which provides automatic updates and reliability.
+
+## Prerequisites
+
+Before manual installation, ensure you have:
+
+- ✅ [Downloaded Piri binary](./download.md) (should be `./piri` in your current directory)
+- ✅ [Configuration file from initialization](./initialization.md) (`config.toml`)
+
+## Installation
+
+### Step 1: Move Binary to System Path
+
+Move the Piri binary to a location in your PATH:
+
+```bash
+# Move to /usr/local/bin (recommended)
+sudo mv piri /usr/local/bin/piri
+
+# Make sure it's executable
+sudo chmod +x /usr/local/bin/piri
+```
+
+### Step 2: Verify Installation
+
+```bash
+# Verify piri is accessible
+piri version
+```
+
+### Step 3: Store Your Configuration
+
+You can store your configuration file in several locations:
+
+**Option A: User config directory (recommended)**
+```bash
+# Piri automatically loads from here
+mkdir -p ~/.config/piri
+cp config.toml ~/.config/piri/config.toml
+```
+
+**Option B: Custom location**
+```bash
+# Store anywhere, but you'll need to specify --config flag
+mkdir -p /etc/piri
+sudo cp config.toml /etc/piri/config.toml
```

Comment:
I just generally wouldn't leave multiple copies of the config file on the system so you don't get confused which one is in use.

```suggestion
sudo mv config.toml /etc/piri/
```